templates/definition/vehicle/tesla.yaml

```
@@ -37,6 +37,10 @@ params:
     advanced: true
   - name: control
     deprecated: true
+  - name: bleProxyBaseUrl
```

proxy

sollte genügen. V.a. muss erklärt werden, was der Parameter tut, sonst versteht das kein Anwender.

---

good point.
```suggestion
  - name: commandProxy
```
to clarify it's only for sending commands and not reading data